app/src/main/java/org/simple/clinic/onboarding/OnboardingScreen.kt

```diff
@@ -93,6 +89,6 @@ class OnboardingScreen(context: Context, attributeSet: AttributeSet) : RelativeL
   }
 
   override fun moveToRegistrationScreen() {
-    screenRouter.clearHistoryAndPush(SelectCountryScreenKey(), RouterDirection.FORWARD)
+    findNavController().navigate(R.id.action_onboardingScreen_to_selectCountryScreen)
```

Comment:
It wouldn't, we are using `popUpTo` & `popUpToInclusive` attrs on the action in the graph, that will handle this behaviour. So if we press back it will close the app rather than opening the previous screen.